<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetitionStartListDisplayItemWidget</class>
 <widget class="QWidget" name="CompetitionStartListDisplayItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>45</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_index">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Quicksand Medium</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: black;</string>
     </property>
     <property name="text">
      <string>1.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Quicksand [PfEd]</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: black;</string>
     </property>
     <property name="text">
      <string>Marius Lindvik</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_flag">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_active">
     <property name="font">
      <font>
       <family>Quicksand Medium</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox{
color: black;
}
QCheckBox::indicator {
    width: 20px;
    height: 20px;
    background-color: gray;
    border-radius: 10px;
    border-style: solid;
    border-width: 1px;
    border-color: black;
}
QCheckBox::indicator:checked {
    background-color: qradialgradient(spread:pad, 
                            cx:0.5,
                            cy:0.5,
                            radius:0.9,
                            fx:0.5,
                            fy:0.5,
                            stop:0 rgba(0, 255, 0, 255), 
                            stop:1 rgba(0, 64, 0, 255));
}</string>
     </property>
     <property name="text">
      <string>Nieaktywny</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
